lib/registrant.rb

```diff
@@ -0,0 +1,27 @@
+class Registrant
+  attr_reader :name, :age, :permit
+  attr_accessor :license_data
+  
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = 
+      {:written=>false, 
+      :license=>false, 
+      :renewed=>false}
+  end
+
+  def permit?
+    if permit == true
```

Comment:
What if the only thing in this method was `@permit`. Would it return what you want?